tests/Core/Components/List/FluentSelectTests.razor

```diff
@@ -45,9 +45,9 @@
         // Arrange and Act
         var cut = Render(
     @<FluentSelect TOption="string">
-        <FluentOption Value="One">One</FluentOption>
-        <FluentOption Value="Two">Two</FluentOption>
-        <FluentOption Value="Three">Three</FluentOption>
+        <FluentOption Value="@("One")">One</FluentOption>
```

Comment:
Why did you update `"One"` to `"@("One")"`?

Reply:
FluentOption is now generic. When using string as the type, you need to specify the values like this. But made some more changes so will check if it is still necessary